### PR TITLE
prusaslicer: Update 2.5.0, checkver, Remove 32bit

### PR DESCRIPTION
**Pull request by b4imetu**
Closes #XXXX

Relates to #XXXX

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by b4imetu** [comment]
@tech189 buddy, I still need your help 🤣

File: bucket/prusaslicer.json [hunk 1]
```diff
@@ -24,17 +19,13 @@
     ],
     "checkver": {
         "url": "https://api.github.com/repos/prusa3d/PrusaSlicer/releases",
-        "regex": "(?sm)PrusaSlicer-[\\d.]+\\+win32-(?<timestamp32>\\d+)\\.zip.*?PrusaSlicer-([\\w.-]+)\\+win64-(?<timestamp64>\\d+)\\.zip"
+        "regex": "PrusaSlicer-([\\w.-]+)\\+win64-(?<timestamp64>\\d+)\\_signed.zip"
```
Comment (issaclin32) [inline bucket/prusaslicer.json:22]:
```suggestion
        "regex": "PrusaSlicer-([\\w.-]+)\\+win64-(?<timestamp64>\\d+)_signed\\.zip"
```

The regex escape `\_` is invalid for the .NET variant.
https://github.com/ScoopInstaller/Extras/runs/8270519843?check_suite_focus=true
```
Line | 
     290 |  …     $regex = New-Object System.Text.RegularExpressions.Regex($regexp) 
         |                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
         | Exception calling ".ctor" with "1" argument(s): "Invalid pattern 
         | 'PrusaSlicer-([\w.-]+)\+win64-(?<timestamp64>\d+)\_signed.zip' at offset 50. Unrecognized escape 
         | sequence \\_." 
```

Comment (b4imetu) [reply bucket/prusaslicer.json]: Thank you very much. I will more about regular expressions

**Pull request merged**